ingest/ingest_pipeline.py

```diff
@@ -493,6 +511,13 @@ def run_ingest(ingest, arguments, parsed_args):
             config.set_parent_event_name("ingest-pipeline:subsample:ingest")
             status_subsample = ingest.subsample()
             status.append(status_subsample)
+    elif "differential_expression" in arguments:
+        if arguments["differential_expression"]:
+            config.set_parent_event_name(
+                "ingest-pipeline:differential_expression:ingest"
+            )
+            status_de = ingest.calculate_de()
+            status.append(status_de)
```

Comment:
The inner condition strikes me as something that ought to be redundant with the outer condition.  

Having `ingest` as the terminal segment of the event name also feels confusing.  In the context of our pipelines, it'd be cool if we could simply say "ingest" to mean "traditionally transform and load SCP study files to MongoDB" and "differential expression" to mean "run DE calculations on previously-validated and ingested data".

Finally, trivially, delimiter should be hyphen instead of underscore.

```suggestion
    elif "differential-expression" in arguments:
        config.set_parent_event_name(
            "ingest-pipeline:differential-expression"
        )
        status_de = ingest.calculate_de()
        status.append(status_de)
```